ajax/libs/victor/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "victor",
+  "description": "A JavaScript 2D vector class with methods for common vector operations",
```

Comment:
@scissorsneedfoodtoo Thanks.
And please add `"version": "1.1.0"` this field.
Because you add v1.1.0's files from upstream. (sorry that I forgot to check this one.)

Except this, I think this PR is almost done.

Many thanks.